packages/formstr-app/src/containers/PollResponse/SingleChoiceOptions.tsx

```diff
@@ -0,0 +1,38 @@
+import { Radio , Typography } from "antd";
+import type { RadioChangeEvent } from "antd";
+const { Text } = Typography;
+
+interface SingleChoiceOptionsProps {
+  options: Array<[string, string, string]>;
+  response: string[];
+  handleResponseChange: (value: string) => void;
+}
+
+export const SingleChoiceOptions: React.FC<SingleChoiceOptionsProps> = ({
```

Comment:
I wish there was a way to reuse the form components for RadioButton here instead of creating a new component, is that possible?